hq_superset/utils.py

```diff
@@ -135,10 +135,7 @@ def sync_domain_role(self, domain):
         The user gets assigned at least 3 roles in order to function on any domain:
         1. hq_user_role: gives access to superset platform
         2. domain_schema_role: restricts user access to specific domain schema
-        3. domain_user_role: restricts access for particular user on domain in accordance with how the permissions
-        are defined on CommCare HQ.
-
-        Any additional roles defined on CommCare HQ will also be assigned to the user.
+        3. Either the Gamma role for "edit" users or the READ_ONLY_ROLE_NAME for "view only" user
```

Comment:
nit: what is an "edit" user? or "view only" user? in terms of CCA

Reply:
https://github.com/dimagi/commcare-analytics/pull/79/commits/13d513caad58b27266b34a072e262e00b4c4a700